### DISCUSSION TITLE
What is the Method to Set Min and Max on the Y-Axis?

### DISCUSSION THREAD

**Discussion by lugan1** [patrykandpatrick/vico#457] [Questions] [answered]
I have searched through all the options to set the min and max of the Y-axis to 20 ~ 40, but I couldn't find it. Is it possible to set the min and max on the Y-axis?

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can do this via `AxisValuesOverrider`. Please see the last paragraph of the introduction [here](https://patrykandpatrick.com/vico/wiki/1.13.0-beta.1/charts).

**Reply by natabma** [reply]
Hi, this link does not work anymore. This is the right one: https://patrykandpatrick.com/vico/wiki/cartesian-charts/layers/#axisvalueoverrider

**Reply by Gowsky** [reply]
Thanks for sharing the updated link. As the API and documentation evolve, links may expire, and comments may become outdated. For the most up-to-date information, we recommend checking the documentation first.
